modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -418,7 +418,7 @@ private function add_accordion_style_section() {
 				'type' => Controls_Manager::DIMENSIONS,
 				'size_units' => [ 'px', '%', 'em', 'rem', 'custom' ],
```

Comment:
Update: --border-bottom-right-radius etc

Reply:
These variables have been removed from _container.scss. This override no longer does anything as far as I can tell. Unless there's some BC angle in play that I'm not seeing.